package.json

```diff
@@ -6,7 +6,9 @@
   "directories": {
     "test": "test"
   },
-  "dependencies": {},
+  "dependencies": {
+    "jasmine-spec-reporter": "^5.0.2"
```

Comment:
this dependency should be installed as dev-dependency

Reply:
Okay, I've made the change